numpy/_core/numeric.py

```diff
@@ -2517,6 +2516,63 @@ def array_equiv(a1, a2):
     return builtins.bool((a1 == a2).all())
 
 
+def _astype_dispatcher(x, dtype, /, *, copy=None):
+    return (x, dtype)
+
+
+@array_function_dispatch(_astype_dispatcher)
+def astype(x, dtype, /, *, copy = True):
+    """
+    Copies an array to a specified data type.
+
+    This function is an Array API compatible alternative to
+    `numpy.ndarray.astype`.
+
+    Parameters
+    ----------
+    x : array_like
```

Comment:
```suggestion
    x : `ndarray`
```

Maybe also a note that array-likes are explicitly not supported here?

Reply:
Right, I updated it to `x : ndarray` to follow existing docs in other functions.
I also added the note.